js/shared_controls.js

```diff
@@ -85,8 +85,8 @@ $(".sl .dvs").keyup(function () {
 function getHPDVs(poke) {
 	return (~~poke.find(".at .dvs").val() % 2) * 8 +
             (~~poke.find(".df .dvs").val() % 2) * 4 +
-            (~~poke.find(gen === 1 ? ".sl .dvs" : ".sa .dvs").val() % 2) * 2 +
-            (~~poke.find(".sp .dvs").val() % 2);
+				(~~poke.find(".sp .dvs").val() % 2) * 2 +
```

Comment:
Remove white space

Reply:
Maybe good enough if I put them all in one line, so it doesn't end up like the first recovery text PR (messy indenting everywhere) ?